codebuild/compliance.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.2
+
+env:
+  variables:
+    NODE_OPTIONS: "--max-old-space-size=4096"
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: latest
+    commands:
+      - npm ci --unsafe-perm
```

Comment:
This file appears to do nothing?
Any static checking (esLint etc, should be done here)

Reply:
I was thinking of doing esLint and prettier in a separate PR because they could change a lot of files and overall be a formatting PR. I only added this compliance file so we can add it later. What are your thoughts? We have lint-staged which gets triggered with husky. Should the linting be done in this PR? If not, I can delete this file and add it later on.

Reply:
Yes, do whatever lint is happening in lint-staged....